scripts/linters/custom_eslint_checks/rules/test-message-style.js

```diff
@@ -0,0 +1,94 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Lint check to ensure that correct style is followed for the
+ * test messages in calling of 'it'.
+ */
+
+'use strict';
+
+module.exports = {
+  meta: {
+    type: 'layout',
+    docs: {
+      description: 'There should not be any unused directive',
+      category: 'Stylistic Issues',
+      recommended: false
+    },
+    fixable: null,
+    schema: [],
+    messages: {
+      useShould: 'Test message should start with \'should\'',
+      singleSpace: 'Do not use multiple consecutive spaces in the test message',
```

Comment:
Please remove multiple consecutive spaces in the test message
Please remove space from the end of the test message

Reply:
Done.